source/documentation/values/calculations.md

```diff
@@ -156,61 +156,119 @@ CSS to unitless numbers:
   @debug calc(-infinity) < math.$min-number  // true
 {% endcodeExample %}
 
-## `min()` and `max()`
+## Calculation Functions
 
-{% compatibility 'dart: ">=1.11.0 <1.42.0"', 'libsass: false', 'ruby: false', 'feature: "Special function syntax"' %}
-  LibSass, Ruby Sass, and versions of Dart Sass prior to 1.11.0 *always* parse
-  `min()` and `max()` as Sass functions. To create a plain CSS `min()` or
-  `max()` call for those implementations, you can write something like
-  `unquote("min(#{$padding}, env(safe-area-inset-left))")` instead.
-
-  Versions of Dart Sass between 1.11.0 and 1.40.0, and between 1.40.1 and 1.42.0
-  parse `min()` and `max()` functions as [special functions] if they're valid
-  plain CSS, but parse them as Sass functions if they contain Sass features
-  other than interpolation, like variables or function calls.
-
-  Dart Sass 1.41.0 parses `min()` and `max()` functions as calculations, but
-  doesn't allow unitless numbers to be combined with numbers with units. This
-  was backwards-incompatible with the global `min()` and `max()` functions, so
-  that behavior was reverted.
-
-  [special functions]: /documentation/syntax/special-functions
+{% compatibility 'dart: "1.64.0"', 'libsass: false', 'ruby: false', 'feature: "Special function syntax"' %}
```

Comment:
The feature tag "special function syntax" is misleading, since we usually use that term to refer to [functions that are parsed as strings](https://sass-lang.com/documentation/syntax/special-functions/). Really what this is describing is the version all the new functions got added in, so maybe "additional functions" is a better name.